src/server/hotkey_collector.cpp

```diff
@@ -168,6 +168,7 @@ inline void hotkey_collector::change_state_by_result()
     case hotkey_collector_state::FINE_DETECTING:
         if (!_result.hot_hash_key.empty()) {
             change_state_to_finished();
+            derror_replica("Find the hotkey: {}", _result.hot_hash_key);
```

Comment:
why did you use derror_replica? what about dwarn_replica?

Reply:
In pegasus, `error` is higher than `dwarn`